Flutter/pdf-viewer/undo-redo.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: Undo and Redo actions on the annotations in Flutter PDF Viewer widget | Syncfusion
+description: Learn here all about Undo and Redo actions on the annotations in Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: Flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Undo and Redo actions on the annotations in Flutter PDF Viewer widget (Syncfusion)
+
+If you performed any undesired actions when on adding, removing or editing annotations, you can undo and redo the action to restore the previous state. This section will go through how to perform the undo and redo the changes made on the annotations.
```

Comment:
If you performed any undesired actions when adding, removing, or editing annotations, you can undo and redo the action to restore the previous state. This section will go through how to undo and redo the changes made to the annotations.